awswrangler/s3/_read_parquet.py

```diff
@@ -867,11 +873,37 @@ def read_parquet_table(
         path: str = location if location.endswith("/") else f"{location}/"
     except KeyError as ex:
         raise exceptions.InvalidTable(f"Missing s3 location for {database}.{table}.") from ex
+    path_root: Optional[str] = None
+    paths: Union[str, List[str]] = path
+    # If filter is available, fetch & filter out partitions
+    # Then list objects & process individual object keys under path_root
+    if partition_filter is not None:
+        available_partitions_dict = _get_partitions(
+            database=database,
+            table=table,
+            catalog_id=catalog_id,
+            boto3_session=boto3_session,
+        )
+        available_partitions = list(available_partitions_dict.keys())
+        if len(available_partitions) > 0:
```

Comment:
Nitpick but would `if available_partitions` do it?

Reply:
Empty list is falsey so yeah that would do. I'll change.